ecl2df/fipreports.py

```diff
@@ -176,13 +176,16 @@ def fill_parser(parser):
         "-o", "--output", type=str, help="Output CSV filename", default="outflow.csv"
     )
     parser.add_argument("-v", "--verbose", action="store_true", help="Be verbose")
+    parser.add_argument("--debug", action="store_true", help="Be even more verbose")
```

Comment:
Be verbosest? Oh, the humor! :)

Reply:
Changed to a duller `Debug mode for logging` ... :)